scripts/load_safe_deployments.py

```diff
@@ -0,0 +1,44 @@
+import json
```

Comment:
Maybe we should move this script and chains one to a common and descriptive folder like `scripts/generators`? What do you think?

Reply:
Agree